docs/runbooks/case-viewer/every-day/adding-a-new-team.md

```diff
@@ -0,0 +1,102 @@
+---
+id: adding-a-new-team
+title: Add a new team
+slug: /runbooks/case-viewer/every-day/adding-a-new-team
+---
+
+# Add a new team 
+
+There are two platforms that include the concept of teams. 
+  *  **case viewer api** (main tool) 
+  * **core pathways tool**.
+
+#### This guide covers
+
+1. How to add a new team to the case viewer api 
+2. How to add a new team to the core pathways tool
+------------------------------------------------------
+
+## 1. How to add a new team to the case viewer api 
+#### Steps:
+  * In order to send a new team request to the API, refer to the Runbook guide on [Connecting to the case viewer API](../development/connect-to-social-care-case-viewer-api.md)
+  * Appending the correct value for Route to the endpoint
+  * Creating a new team JSON object
+  * Making a request to the API using CURL
+  * Checking it worked
+### -- Appending the correct value for Route to the endpoint --
+
+Once you have the correct endpoint for hitting either Staging or Production, if you wish to direct your HTTP request to the right resource in the API then you will have to append the correct route to the endpoint.
+
+Route information can be found in the specific controller that handles the resource you wish to target.
+In the Social Care Case Viewer API, the controller for teams is found at:
+
+`social-care-case-viewer-api/SocialCareCaseViewerApi/V1/Controllers/TeamController.cs`
+
+Route is defined in that file like so **[Route("api/v1/teams")]**
+
+Append ```api/v1/teams``` to the endpoint in order to target the teams resource e.g.,
+```
+https://<api gateway id>.execute.<aws region>.amazonaws.com/<environment>/api/v1/teams
+```
+
+--------------------------------------------------------
+
+### -- Creating a new JSON team object --
+
+The API expects to receive a JSON object containing the data properties needed to create a new team.
+
+These properties consist of a `name` and a `context`
+
+name = the name of the new team
+
+context = refers to which service, 
+
+“A” for ASC or “C” for CFS
```

Comment:
Sorry to push back here, but can you either use the `abbr` tag here to explain the acronyms, or better yet get rid of them all together.

For reference https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr

Reply:
Removed